src/outcome_spaces/dispersion.jl

```diff
@@ -129,7 +129,7 @@ function outcome_space(est::Dispersion)
     c, m = 1:est.c, est.m
     cart = CartesianIndices(ntuple(i -> c, m))
     V = SVector{m, Int}
-    return map(i -> V(Tuple(i)), vec(cart))
+    return sort(map(i -> V(Tuple(i)), vec(cart)))
```

Comment:
`sort!` instead: we don't need the extra copy

Reply:
Fixed.